grpc-ballerina/error_codes.bal

```diff
@@ -17,36 +17,36 @@
 //Package defines the canonical error codes used by gRPC.
 
 # The gRPC error status code: 0 OK.
-public const int OK = 0;
```

Comment:
Shouldn’t these error codes need to be public?

Reply:
Since we allow passing specific error values now, AFAIK these error code constants are not used anymore. We can actually remove it. is there any place you remember